### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by kanybekasanbekov** [hysts/pytorch_resnet_preact#1]
According to your results, I think your model clearly overfits. Am I right? Have you tried anything to reduce overfitting?

**Comment by hysts** [comment]
Hi, @kanybekasanbekov 

So you mean you think the model is overfitting because the test loss is increasing after the first lr reduction? I'm not sure if we should think of it as overfitting since the test accuracy is still improving.
Anyway, stronger augmentation and cosine annealing can reduce the increase of test loss. See [this repo](https://github.com/hysts/pytorch_image_classification).

**Comment by kanybekasanbekov** [comment]
I have two reasons why the model overfits:
1. Test Loss increases
2. There is a gap between Train/Test Errors(Losses), see the image below.
![ResNet-preact-110_basic](https://user-images.githubusercontent.com/34956058/120736690-ab1af880-c527-11eb-9c92-c0fc8efe6651.png)
Is this gap the sign of overfitting? I am facing this kind of gap when training PreActResnet on the Imagenet dataset.

**Comment by hysts** [comment]
@kanybekasanbekov 

Could be. But it's common for test loss to be higher than training loss.
I'm not sure why val loss increases in case of classification tasks, but maybe it's because cross entropy loss could become higher if the model outputs a high confidence prediction for a class similar to the target class. So as long as the target metric, i.e. accuracy, is improving, I don't think we have to worry too much about it.

**Comment by kanybekasanbekov** [comment]
Thanks for your reply. When I train Original Resnet val and train accuracies(losses) are almost the same but in the case of PreActResnet train accuracy(loss) becomes much higher(lower). So I suppose my PreActResnet is overfitting. I am gonna investigate more about it. Anyway, I appreciate your answer.
